packages/client/src/devices/SpeakerManager.ts

```diff
@@ -38,6 +38,11 @@ export class SpeakerManager {
    * @returns an Observable that will be updated if a device is connected or disconnected
    */
   listDevices() {
+    if (isReactNative()) {
```

Comment:
Can you also add a note in the JS doc similar like for `.select()` as well?

Reply:
Fixed 👍🏻